core/model.py

```diff
@@ -46,7 +62,13 @@ def _sample(self, preds, temperature=.6):
         probas = np.random.multinomial(1, preds, 1)
         return np.argmax(probas)
 
-    def predict(self, sentence, gen_chars=50):
+    def _pre_process(self, args_dict):
```

Comment:
These methods have default implementations - if you're not actually overriding them, I don't see the value in having this here.

Reply:
The reason I put these here was to be consistent with other models, but I have removed them in the next update.